frontend/src/modules/Home/Components/Home.tsx

```diff
@@ -35,18 +36,17 @@ export const Home = () => {
             <StyledWelcomeText>Welcome to Zing!</StyledWelcomeText>
           </StyledTitleWrapper>
           <StyledButtonsWrapper>
-            <WhiteButton
-              label="Log In"
-              onClick={() => {
-                history.push('/login')
-              }}
-            />
-            <PrimaryGradientButton
-              label="Sign Up"
-              onClick={() => {
-                history.push('/signup')
-              }}
-            />
+            <Button
+              color="secondary"
+              variant="outlined"
+              component={Link}
+              to="/login"
```

Comment:
This is the better way of doing it, IMO

Reply:
Are you referring to how it was with "push history" before? lol

Reply:
New way is better